src/frontend/typer/fagsak.ts

```diff
@@ -7,6 +7,14 @@ export enum Behandlingstype {
     REVURDERING = 'REVURDERING',
 }
 
+// Enum
+export enum VedtakResultat {
+    INNVILGET = 'INNVILGET',
```

Comment:
Use this to set the result instead of hard coded values in the radio button group 🌮

Reply:
ok